src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return 0.5 * firstLeg * secondLeg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: right triangle, area: "
+                + getArea() + " sq. units, firstLeg: "
+                + firstLeg + " units, secondLeg: "
+                + secondLeg + " units, color: "
+                + color);
```

Comment:
Consider using String.format for better readability when constructing the output string in the draw() method.